gdal/apps/gdalwarp_lib.cpp

```diff
@@ -2791,6 +2822,14 @@ GDALWarpCreateOutput( int nSrcCount, GDALDatasetH *pahSrcDS, const char *pszFile
             GCI_AlphaBand );
     }
 
+/* -------------------------------------------------------------------- */
+/*      Copy the raster attribute table, if required.                   */
+/* -------------------------------------------------------------------- */
+    if( hRAT != nullptr )
+    {
+        GDALSetDefaultRAT( GDALGetRasterBand(hDstDS,1), hRAT );
```

Comment:
when warping to a non-RAT / non-PAM enabled format, the above could trigger a warning if the base GDALRasterBand::SetDefaultRAT() is called. Perhaps surround this by CPLPushErrorHandler(CPLQuietErrorHandler) / CPLPopErrorHandler()

Reply:
https://github.com/OSGeo/gdal/blob/master/gdal/gcore/gdalrasterband.cpp#L5877-L5879 currently reports a "SetDefaultRAT() not implemented for this format."